docs/hpc/pilot/cx3-phase2.md

```diff
@@ -157,6 +166,43 @@ The following table provides an explanation of what each directive means in the
 
 The general advice it to set the resources to what is needed for a job but to not over specify. For example, don't set CPU type if your job can run fine or Rome or Icelake. This ensures your job starts as early as possible.
 
+### Job Sizing Guidance
+
+The following queues of jobs are supported:
+
+| Queue | Use Cases | Nodes per job | No. of cores per job<br>(ncpus) | Mem<br>(GB) | Walltime<br>(hrs) |
+| ----- | --------- | :-------------: | ---------------------------- | -------- | -------------- |
+| small24 | Low core jobs 24h | 1 | 1 - 16 | 1 - 128 | 0 - 24 |
+| small72 | Low core jobs 72h | 1 | 1 - 16 | 1 - 128 | 24 - 72 |
+| [jupyter](#jupyter) | Queue for JupyterHub jobs* | 1 | 1, 4, 8 | 8, 32, 64 | 2, 4, 8 |
+| [jupytergpu](#jupytergpu) | Queue for JupyterHub GPU jobs* | 1 | 4 | 32 | 8 |
+| medium24 | Single-node jobs 24h | 1 | 1 - 64 | 1 - 450 | 0 - 24 |
+| medium72 | Single-node jobs 72h | 1 | 1 - 64 | 1 - 450 | 24 - 72 |
+| large24 | Whole node jobs 24h | 1 | 1 - 128 | 1 - 920 | 0 - 24 |
+| large72 | Whole node jobs 72h | 1 | 1 - 128 | 1 - 920 | 24 - 72 |
+| largemem72 | Large memory jobs | 1 | 1 - 128 | 921 - 4000 | 0 - 72 |
+| gpu72 | Main queue for gpu jobs* | 1 | 1 - 128 | 1 - 920 | 0 - 72 |
+| capability24 | Multi-node jobs 24h | 2 - 4 | 1 - 256 | 1 - 2048 | 0 - 24 |
+| capability48 | Multi-node jobs 48h | 2 - 4 | 1 - 256 | 1 - 2048 | 24 - 48 |
+
+\* Please see details for specific queues below as there may be additional restrictions or limitations.
+
+#### jupyter
+This queue is where JupyterHub jobs are run. There is a limit of 1 concurrent job per user across both jupyter queues.
+
+#### jupytergpu
+This queue has NVIDIA A40 (46GB) GPU's. There is a limit of 1 concurrent job per user across both jupyter queues.
```

Comment:
```suggestion
This queue has NVIDIA A40 (48GB) GPU's. There is a limit of 1 concurrent job per user across both jupyter queues.
```

Reply:
Really minor fix that I didn't spot the first time.

Reply:
Very well spotted